api/python/api/cli.py

```
@@ -124,39 +130,36 @@ def create_db():
 
 
-@app.command()
-def read_fit(
-    fit_file: str = typer.Argument(),
-    out_file: str = typer.Option(None),
+async def _read_fit_async(
+    fit_file: str,
+    out_file: str | None = None,
 ):
     """Read a .fit file and parse activity, laps, and tracepoints."""
     locations = json.load(open("./data/locations.json")).get("locations")
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)